PWGHF/D2H/Tasks/taskCharmPolarisation.cxx

```diff
@@ -605,7 +605,7 @@ struct TaskPolarisationCharmHadrons {
         massDau = massPi; // (*)
         const float bkgRotAngle = bkgRotationAngleStep * bkgRotationId;
         std::array<float, 3> threeVecSoftPi{candidate.pxSoftPi() * std::cos(bkgRotAngle) - candidate.pySoftPi() * std::sin(bkgRotAngle), candidate.pxSoftPi() * std::sin(bkgRotAngle) + candidate.pySoftPi() * std::cos(bkgRotAngle), candidate.pzSoftPi()}; // we rotate the soft pion
-        std::array<float, 3> threeVecD0Prong0{candidate.pxProng0(), candidate.pxProng0(), candidate.pzProng0()};
+        std::array<float, 3> threeVecD0Prong0{candidate.pxProng0(), candidate.pyProng0(), candidate.pzProng0()};
```

Comment:
You can avoid these typos by using the `pVector(...)` columns. See https://github.com/AliceO2Group/O2Physics/pull/5710
```suggestion
        std::array<float, 3> threeVecD0Prong0{candidate.pVectorProng0()};
```

Reply:
Same for `Prong1`.